client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx

```diff
@@ -0,0 +1,141 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	headline,
+	palette,
+	space,
+	textSans,
+} from '@guardian/source-foundations';
+import { Button, Stack } from '@guardian/source-react-components';
+import { Link } from 'react-router-dom';
+import { benefitsCss } from '@/client/components/mma/shared/benefits/BenefitsStyles';
+import { stackedButtonLayoutCss } from '@/client/styles/ButtonStyles';
+
+function GreyBulletpoints() {
```

Comment:
```suggestion
function GreyBulletpoint() {
```
Maybe should be singular, for clarity?